tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -116,6 +116,7 @@ static char *fn_sig_ptr = NULL;
 static void insert_entry_label(int);
 static void insert_jump_entry_instr(int);
 static void store_return_value_for_entry(OPERAND *, int);
+static void insertLLVMDbgValue(OPERAND *, LL_MDRef, SPTR, LL_Type *);
```

Comment:
Using camelCase here breaks the pattern that other functions use (underscore naming).

Reply:
It is an existing function so did not notice earlier. Thanks for pointing it out. Correcting it now.